app/assets/stylesheets/application.css

```diff
@@ -91,36 +106,69 @@ body {
   grid-template-columns: 1fr 1fr 1fr 1fr;
   align-items: center;
   height: 4rem;
-  border: 0.1rem solid red;
+  margin-top: 1rem;
+  border: 0.1rem solid blue;
+  padding: 0.5rem;
+}
+
+.recipe-food-action {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  align-items: baseline;
 }
 
 .recipe-food-header {
-  border: 0.1rem solid red;
+  border: 0.1rem solid blue;
 }
 
-.public-recipe-item,
-.shopping-list-header {
+.public-recipe-item {
+  border: 0.1rem solid blue;
+  border-radius: 1rem;
+  padding: 1rem 0.2rem;
   display: grid;
   grid-template-columns: 1fr 3fr;
   gap: 2rem;
   margin: 2rem 0;
 }
 
+.shopping-list-header {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 1rem;
+  margin: 2rem 0;
+}
+
 .shopping-list-item {
   display: grid;
   grid-template-columns: 1fr 1fr 3fr;
 }
 
-.recipe-food-action {
+.recipe-food-a {
+  border: 0.1rem solid;
+  padding: 0.1rem 0.3rem;
+  text-decoration: none;
+  color: black;
+}
+
+.shopping-and-add-new-ingredient {
   display: flex;
   flex-direction: row;
```

Comment:
[OPTIONAL] You don't need `flex-direction: row` since row is the default.

Reply:
Oh, thank you for this idea, @fobadara